lib/matplotlib/axes/_axes.py

```diff
@@ -384,6 +402,208 @@ def legend(self, *args, **kwargs):
     def _remove_legend(self, legend):
         self.legend_ = None
 
+    def inset_axes(self, rect, transform=None, **kwargs):
+        """
+        Add a child inset axes to this existing axes.
+
+        Parameters
+        ----------
+
+        rect : [x0, y0, width, height]
+            Lower-left corner of inset axes, and its width and height.
+
+        transform : `.Transform`
+            Defaults to `ax.transAxes`, i.e. the units of *rect* are in
+            axes-relative coordinates.
+
+        zorder : number
+            Defaults to 5 (same as `.Axes.legend`).  Adjust higher or lower
+            to change whether it is above or below data plotted on the
+            parent axes.
+
+        Other Parameters
+        ----------------
+
+        kwargs are passed on to the `axes.Axes` child axes.
+
+        Returns
+        -------
+
+        New `.axes.Axes` instance.
+
+        Examples
+        --------
+
+        This example makes two inset axes, the first is in axes-relative
+        coordinates, and the second in data-coordinates::
+
+            fig, ax = plt.suplots()
+            ax.plot(range(10))
+            axin1 = ax.inset_axes([0.8, 0.1, 0.15, 0.15])
+            axin2 = ax.inset_axes([5, 7, 2.3, 2.3], transform=ax.transData)
+
+        """
+        if transform is None:
+            transform = self.transAxes
+        label = kwargs.pop('label', 'inset_axes')
```

Comment:
would tend to make this and zorder keyword-only args (see my kwonly-args PR for example).  again, not a big deal.

Reply:
Ah, I see why I ignored you here - this is pretty clear kwarg passing down to axes.  I don't want to pull individual kwargs out for documentation reasons that are just passed down, but on the other hand I do want to set the default to something different than what Axes would set it to by itself.  Moreover, 99% of the time, no one wants to define the label...